tools/test-network-impl.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+if ! "$CHUTNEY_PATH/tools/bootstrap-network.sh" "$NETWORK_FLAVOUR"; then
+    CHUTNEY_WARNINGS_IGNORE_EXPECTED=false CHUTNEY_WARNINGS_SUMMARY=false \
+        "$WARNING_COMMAND"
+    "$WARNINGS"
+    $ECHO "bootstrap-network.sh failed"
+    exit 1
+fi
+
+# chutney starts verifying after 20 seconds, keeps on trying for 60 seconds,
```

Comment:
This comment is now wrong, and that's probably my fault from a previous patch:
```suggestion
# chutney starts verifying after START seconds, keeps on trying for BOOTSTRAP seconds,
```

Reply:
I opened 
https://trac.torproject.org/projects/tor/ticket/29760 to fix this issue.